kemeny_sourcemash.tex

```diff
@@ -35,7 +35,9 @@
 \pagebreak
 \section{Project Overview}
 
-Sourcemash 
+Sourcemash is a web-based application that categorizes news articles to help users read their RSS feeds faster. Sourcemash achieves this goal is three steps. First, users select their sources from the Browse page, which offers a pre-seeded list of popular feeds and a form allowing manual input of new RSS feeds; Sourcemash is designed to categorize your news, not someone else's. Second, Sourcemash's categorization engine analyzes each article in the user's subscribed feeds to find the categories that best match; our algorithm is constantly working in the background, so users can find articles in the topics they're looking for faster. Finally, Sourcemash offers the ability to read thew news by both category and feed, offering users the ability to integrate their news reading preferences into the application experience. 
```

Comment:
I don't know how I feel about the "three steps" approach to describing the site for the judges. We're not trying to sell them on the site, but of it's purpose. I think we can simplify this section to just explain that Sourcemash is an RSS feed reader that algorithmically categorizes articles so that you can sort by subject.